docs/reference/vpc-association-policy.md

```diff
@@ -0,0 +1,85 @@
+1. Update the VPCAssociationPolicy with AssociateWithVpc is false and empty security group ids
+2. Update the VPCAssociationPolicy with AssociateWithVpc is true and empty security group ids
+
+Be cautious to set AssociateWithVpc to false. That can break traffic from the current cluster workloads to the gateway.
+
+
+| Field	                                                                                                                      | Description	                                        |
+|-----------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------|
+| `apiVersion` *string*	                                                                                                      | ``application-networking.k8s.aws/v1alpha1`` 	       |
+| `kind` *string*	                                                                                                            | ``VpcAssociationPolicy``                            |
+| `metadata` [*ObjectMeta*](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#objectmeta-v1-meta)        	 | Kubernetes metadata for the resource.               |
+| `spec` *VpcAssociationPolicySpec*	                                                                                          | Defines the desired state of VpcAssociationPolicy.	 |
+
+
+## VpcAssociationPolicySpec
+
+Appears on: VpcAssociationPolicy
+
+VpcAssociationPolicySpec defines the desired state of VpcAssociationPolicy.
+
+
+
+| Field	                                                                                                     | Description                                                                                                                                                                                                                |
+|------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `targetRef` *[PolicyTargetReference](https://gateway-api.sigs.k8s.io/geps/gep-713/#policy-targetref-api)*	 | TargetRef points to the kubernetes `Gateway` resource that will have this policy attached. This field is following the guidelines of Kubernetes Gateway API policy attachment.                                             |
+| `associateWithVpc` *bool*	                                                                                 | (Optional)  This field indicates whether targetRef Gateway associate with current k8s cluster VPC. the gateway api controller by default set associateWithVpc to true if this field is not defined in VpcAssociationPolicy |
```

Comment:
"This field" is redundant.

"Indicates whether the `targetRef` Gateway is associated with the current k8s cluster VPC. By default, the Gateway API controller sets this to true if not defined in `VpcAssociationPolicy`."